crithit.lua

```diff
@@ -0,0 +1,22 @@
+--Turns the previously readied attack into a super powerful critical hit.
```

Comment:
Maybe `critical-hit`

Reply:
Also, I can't get this one to work. I'm trying it from multiple places in the `A` menu in adventure mode, but that might not be right.